src/axom/sidre/core/SidreDataTypeIds.h

```diff
@@ -19,7 +19,7 @@
 
 #include <stdint.h> /* for int64_t */
 
-using SIDRE_IndexType = int64_t;
+typedef int64_t SIDRE_IndexType;
```

Comment:
Are we still planning to configure `SIDRE_IndexType` to be `axom::IndexType` as discussed in https://github.com/LLNL/axom/issues/43 ? 
If so, would that be in a follow-up PR?

Reply:
That's still the plan.  There are pending Shroud changes required first.